### PR TITLE
feature: Add new func SessionBuilder::compression to control compression

### PR DESCRIPTION
**Pull request by NobodyXu**
Signed-off-by: Jiahao XU <Jiahao_XU@outlook.com>

### CODE REVIEW

**Comment by jonhoo** [comment]
This change is [Reviewable](https://reviewable.io/reviews/jonhoo/openssh-rs/37)

File: src/builder.rs [hunk 1]
```diff
@@ -137,6 +139,23 @@ impl SessionBuilder {
...
+    /// connections).
+    ///
+    /// NOTE that the ssh server can forcibly disable the compression.
+    ///
+    /// By default, ssh uses configure value set in `~/.ssh/config`.
+    ///
+    /// If `~/.ssh/config` does not enable compression, then it is disabled
+    /// by default.
+    ///
+    /// Defaults to `None`.
```
Comment (jonhoo) [inline src/builder.rs:153]: This last point doesn't make sense I don't think? You already say what the default is above. There is no `Option` involved here.

File: src/builder.rs [hunk 2]
```diff
@@ -137,6 +139,23 @@ impl SessionBuilder {
         self
     }
 
+    /// Enable or disable compression (including stdin, stdout, stderr, data
+    /// for forwarded TCP and unix-domain connections, sftp and scp
+    /// connections).
+    ///
+    /// NOTE that the ssh server can forcibly disable the compression.
```
Comment (jonhoo) [inline src/builder.rs:146]:
```suggestion
    /// Note that the ssh server can forcibly disable the compression.
```

**Review by jonhoo — CHANGES_REQUESTED** [review]
Just a few minor notes!

File: src/builder.rs (hunk 1 above)
Comment (NobodyXu) [reply src/builder.rs]: Fixed!

**Review by jonhoo — APPROVED** [review]
Thanks!

**Pull request merged**